<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionWindow</class>
 <widget class="QMainWindow" name="ConnectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(64, 64, 64)</string>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>371</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);font: 18pt &quot;Papyrus&quot;;</string>
    </property>
    <property name="title">
     <string>TableManager Connection Settings</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>291</width>
       <height>228</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 255, 255);font: 18pt &quot;Papyrus&quot;;</string>
        </property>
        <property name="text">
         <string>TableManager IP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ipAddr"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 255, 255);font: 18pt &quot;Papyrus&quot;;</string>
        </property>
        <property name="text">
         <string>Port no:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="portNo"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 255, 255);font: 18pt &quot;Papyrus&quot;;</string>
        </property>
        <property name="text">
         <string>NickName:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nickName"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="styleSheet">
         <string notr="true">color:rgb(0, 0, 0);font: 18pt &quot;Papyrus&quot;;background-color:rgb(255,255,255)</string>
        </property>
        <property name="text">
         <string>Get Table Details</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>361</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);font: 18pt &quot;Papyrus&quot;;</string>
    </property>
    <property name="title">
     <string>Available Table List</string>
    </property>
    <widget class="QListWidget" name="tableListWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>31</y>
       <width>351</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);font: 18pt &quot;Papyrus&quot;;</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);font: 24pt &quot;Papyrus&quot;;</string>
    </property>
    <property name="text">
     <string>Multiplayer BlackJack</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
